src/typings/common.tsx

```diff
@@ -73,3 +73,11 @@ export interface DatabaseEntityBase {
     modifiedByName: string;
     versionId: number;
 }
+
+export interface FaramValues {
+    [key: string]: string | undefined | string[] | FaramValues;
+}
+
+export interface FaramErrors {
+    [key: string]: string | undefined | string [] | FaramErrors;
+}
```

Comment:
This is weird.

Reply:
I believe this works for now

Reply:
can we just not use type `object`? This is misleading at best.